doc/Tutorials.xml

```diff
@@ -383,7 +383,7 @@ Edit <File>PackageName/makedoc.g</File> as follows.
   In the <Code>entities</Code> record enter any entities 
   previously stored in <File>manual.xml</File>. 
   (Again, if you have none, you may safely delete this record.) 
-  To illustrate this option the &AutoDoc; file <File>makedoc.g</File> 
+  To illustrate this option the &AutoDoc; file <File>PackageInfo.g</File> 
```

Comment:
Thank you for your patch, which is much appreciated. Unfortunately it is not quite ready to be merged as-is:

Just above the fold you can see `<Index Key="Entities record in makedoc.g"></Index>`, so this block of text really is about `makedoc.g`.

At some point I've added the ability to *also* specify entities in `PackageInfo.g`, and carelessly removed the code for doing so from `makedoc.g` in ace699ca7ac02cd1980192e6babd84d295212d56 as I had forgotten about this mention in the manual... There's a limit how many techniques one can demonstrate in one package ;-).


The thing is, all these records can be specified in either `PackageInfo.g` or `makedoc.g`, and which is preferred depends on the package authors. I guess the text should explain these choices, and perhaps also show code snippets inline... I can try to look into doing that when I find time -- or, well, of course a PR by someone else would also be fine :-).

So I won't merge this PR as-is, but there definitely is something to be fixed and improved here. So thank you very much for drawing my attention to this.